bosh/releases/pre_render_scripts/singleton-blobstore/blobstore/patch_blobstore_pre-start.sh

```diff
@@ -7,23 +7,29 @@ set -o errexit -o nounset
 target="/var/vcap/all-releases/jobs-src/capi/blobstore/templates/pre-start.sh.erb"
 
 patch --binary --unified --verbose "${target}" <<'EOT'
-@@ -20,14 +20,10 @@
-
+@@ -9,7 +9,7 @@
+   local data_dir=/var/vcap/data/blobstore
+   local store_tmp_dir=$store_dir/tmp/uploads
+   local data_tmp_dir=$data_dir/tmp/uploads
+-  local nginx_webdav_dir=/var/vcap/packages/nginx_webdav
++  local packages_dir=/var/vcap/packages
+ 
+   mkdir -p $run_dir
+   mkdir -p $log_dir
+@@ -19,12 +19,12 @@
+   mkdir -p $data_tmp_dir
+ 
    chown vcap:vcap $store_dir
-   local dirs="$run_dir $log_dir $store_tmp_dir $data_dir $data_tmp_dir $nginx_webdav_dir ${nginx_webdav_dir}/.."
+-  local dirs="$run_dir $log_dir $store_tmp_dir $data_dir $data_tmp_dir $nginx_webdav_dir ${nginx_webdav_dir}/.."
 -  local num_needing_chown=$(find $dirs -not -user vcap -or -not -group vcap | wc -l)
--
--  if [ $num_needing_chown -gt 0 ]; then
--    echo "chowning ${num_needing_chown} files to vcap:vcap"
++  local dirs="$run_dir $log_dir $store_tmp_dir $data_dir $data_tmp_dir $packages_dir"
++  local num_needing_chown="0"
+ 
+   if [ $num_needing_chown -gt 0 ]; then
```

Comment:
_mind blown_ :boom: 
If I'm reading this all correctly, we never chown anything. Therefore, I think we should just nuke this whole thing.

Reply:
Yes, this was my intent when testing.
This won't work when buildpacks are used (I think).